### ISSUE TITLE
Windows Build of Tensorflow, CMakeLists.txt error

### ISSUE THREAD

**Issue by apiszcz** [tensorflow/tensorflow#17071]
The Cmake error is displayed when check for native architecture. The following change reports success for the same test on Windows.

```
-- Performing Test COMPILER_OPT_ARCH_NATIVE_SUPPORTED
-- Performing Test COMPILER_OPT_ARCH_NATIVE_SUPPORTED - Success
```

```
:: EDIT %CMAKED%\CMakeLists.txt to replace -march=native with /arch:AVX2 or if older CPU with one of [AVX2|FMA|SSE4.2|FPMATH|MMX] 
if (tensorflow_OPTIMIZE_FOR_NATIVE_ARCH)
  include(CheckCXXCompilerFlag)
  CHECK_CXX_COMPILER_FLAG("/arch:AVX2" COMPILER_OPT_ARCH_NATIVE_SUPPORTED)
  if (COMPILER_OPT_ARCH_NATIVE_SUPPORTED)
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /arch:AVX2")
  endif()
endif()

```

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by apiszcz** [comment]
:: - MSVC Community 2015
:: - ANACONDA 4.4.4 (Python 3.5.5)
:: - CMake 3.10.2
:: - SWIG 3.0.12
:: - GIT 2.15.1.windows.2
:: - NVIDIA CUDA 9.1, CUDNN 7.05
:: No BAZEL
:: TensorFlow version (latest from GIT repository)

**Comment by bignamehyp** [comment]
I apologize but I am having a hard time understanding what the problem is, and where the problem is. Can you describe this issue in more details?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by apiszcz** [comment]
Please close, Tensorflow with GPU is not supported on Windows.

**Comment by kaufManu** [comment]
@apiszcz I think compiling Tensorflow from source on Windows is not _officially_ supported, but you can make it work nontheless. There are some instructions in the [README file](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/cmake) how to enable GPU support and here is a nice [tutorial as well](https://joe-antognini.github.io/machine-learning/windows-tf-project).

**Comment by apiszcz** [comment]
Has anyone compiled TensorFlow 1.5 or 1.6 on Windows 10 with GPU support?

**Comment by kaufManu** [comment]
I did, although the compilation time was huge (something like 18 hours). However, I do have some problems with using it in another C++ project (check this [StackOverflow post](https://stackoverflow.com/questions/49496758/cannot-link-to-self-built-tensorflow-on-windows-with-visual-studio-when-wholear)), but I don't know whether this is related to how I compiled it. I was able to successfully run some provided example scripts.
